source/documentation/doc-Technical_Reference/topics/Port_Mirroring.adoc

```diff
@@ -9,7 +9,7 @@ The only traffic copied is internal to one logical network on one host. There is
 Port mirroring is enabled or disabled in the vNIC profiles of logical networks, and has the following limitations:
 
 
-* Hot plugging vNICs with a profile that has port mirroring enabled is not supported.
+* Hot linking vNICs with a profile that has port mirroring enabled is not supported.
```

Comment:
What is "hot linking"? I have heard of hot swapping and hot plugging but this term is new to me.

Reply:
I can confirm that this is the only occurrence of "Hot linking vNICs", so if you are sure that the change is correct, it's fine with me.